tests/unit/data_mappers/test_data_mappers.py

```diff
@@ -559,6 +559,26 @@ def test_it_gets_glue_table_format_info():
     ) == handlers.get_glue_table_format(get_table_stub())
 
 
+def test_it_gets_glue_table_parametersless():
+    table = {
+        "Table": {
+            "StorageDescriptor": {
+                "Location": "s3://bucket/",
+                "InputFormat": "org.apache.hadoop.mapred.TextInputFormat",
+                "OutputFormat": "org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat",
+                "SerdeInfo": {
+                    "SerializationLibrary": "org.openx.data.jsonserde.JsonSerDe"
```

Comment:
Should this not be omitted to test the code change?

Reply:
Updated